packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -47,6 +47,12 @@ export interface IButtonProps extends React.HTMLAttributes<HTMLButtonElement | H
   checked?: boolean;
 
   /**
+   * When true, and an icon is passed in, the icon will be hidden until button is checked.
+   * @default false
+   */
+  canCheck?: boolean;
```

Comment:
Instead can we make it that you should set checked to true or false instead of adding this new property? Adding @jspurlin who implemented checked.